### PR TITLE
change the look and feel when there are no images at the screen

### PR DESCRIPTION
**Pull request by mgold1234**
# Description

this commit fixes the empty state when there are no images at the screen,

Fixes # related to https://issues.redhat.com/browse/THEEDGE-3478
Screenshot 2023-07-23 at 18 05 42

## Type of change

What is it?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `npm run lint:js:fix` to check that my code is properly formatted

### CODE REVIEW

**Review by acosferreira — APPROVED** [review]
lgtm

File: src/Routes/ImageManager/ImageSetsTable.js [hunk 1]
```diff
@@ -210,21 +210,32 @@ const ImageTable = ({
...
+              'address changes and keep the operating system unchanged. Image updates are quick, ' +
+              'and rollbacks are easy.',
+          ]}
           secondaryActions={[
             {
               type: 'link',
-              title: 'RHEL for Edge image documentation',
-              link: 'https://access.redhat.com/documentation/en-us/edge_management/2022/html/create_rhel_for_edge_images_and_configure_automated_management/index',
+              title: 'Learn more about OSTree.',
+              link: 'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/composing_installing_and_managing_rhel_for_edge_images/index#introducing-rhel-for-edge-images_composing-installing-managing-rhel-for-edge-images',
```
Comment (loadtheaccumulator) [inline src/Routes/ImageManager/ImageSetsTable.js:225]: Is this the correct link?

File: src/Routes/ImageManager/__snapshots__/ImageSetsTable.test.js.snap [hunk 2]
```diff
@@ -32,46 +32,97 @@ exports[`ImageSets table to render correctly 1`] = `
...
+        <div
+          class="pf-l-stack__item"
+        >
+          <a
+            aria-disabled="false"
+            class="pf-c-button pf-m-link"
+            data-ouia-component-id="OUIA-Generated-Button-link-1"
+            data-ouia-component-type="PF4/Button"
+            data-ouia-safe="true"
+            href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/composing_installing_and_managing_rhel_for_edge_images/index#introducing-rhel-for-edge-images_composing-installing-managing-rhel-for-edge-images"
```
Comment (loadtheaccumulator) [inline src/Routes/ImageManager/__snapshots__/ImageSetsTable.test.js.snap:61]: Is this the correct link?

File: src/Routes/ImageManager/__snapshots__/ImageSetsTable.test.js.snap [hunk 3]
```diff
@@ -32,46 +32,97 @@ exports[`ImageSets table to render correctly 1`] = `
...
-          />
-        </svg>
-      </a>
+          <a
+            aria-disabled="false"
+            class="pf-c-button pf-m-link"
+            data-ouia-component-id="OUIA-Generated-Button-link-2"
+            data-ouia-component-type="PF4/Button"
+            data-ouia-safe="true"
+            href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/composing_installing_and_managing_rhel_for_edge_images/index#introducing-rhel-for-edge-images_composing-installing-managing-rhel-for-edge-images"
```
Comment (loadtheaccumulator) [inline src/Routes/ImageManager/__snapshots__/ImageSetsTable.test.js.snap:105]: Is this the correct link?

**Review by loadtheaccumulator — CHANGES_REQUESTED** [review]
Checking source of the links. I'm not sure they should be RHEL 8.

File: src/Routes/ImageManager/__snapshots__/ImageSetsTable.test.js.snap (hunk 3 above)
Comment (loadtheaccumulator) [reply src/Routes/ImageManager/__snapshots__/ImageSetsTable.test.js.snap:105]: The reason I ask about the doc URL is the current version of the OS RHEL 9, and these links map to RHEL 8.

**Comment by acosferreira** [comment]
> Checking

The link to RHEL9 seems be https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/composing_installing_and_managing_rhel_for_edge_images/index#introducing-rhel-for-edge-images_composing-installing-managing-rhel-for-edge-images

Can we check with doc team?

**Review by loadtheaccumulator — APPROVED** [review]

**Pull request merged**